### PR TITLE
Add dj-lightbox w/ git auto-update

### PR DESCRIPTION
**Pull request by danyg790**

### CODE REVIEW

File: packages/d/dj-lightbox.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "images"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/danyg790/dj-lightbox.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```
Comment (MattIPv4) [inline packages/d/dj-lightbox.json:21]: Please avoid using glob stars, as these can lead to files being included in the CDN that were not intended. You should ideally be specifying wildcards with extensions, for example `*.js`, or `*.@(css|js)`

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/d/dj-lightbox.json (hunk 1 above)
Comment (danyg790) [reply packages/d/dj-lightbox.json]: I have made the change to eliminate the glob stars and set it to include only the js and css files

**Comment by danyg790** [comment]
I have made the change to eliminate the glob stars and set it to include only the js and css files

**Comment by danyg790** [comment]
Why am I getting the "Error: no version found on git", and how would I fix that?

**Pull request closed without merging**